keyboards/ut472/config.h

```diff
@@ -19,9 +19,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x0000
-#define DEVICE_VER      0x0001
+#define VENDOR_ID       0xa103
```

Comment:
```suggestion
#define VENDOR_ID       0xA103
```

Reply:
the VIA codebase has ai03's VENDOR_ID set as 0xa103

is there a reason it is suggested to be caps, here?

Reply:
For consistency. The USB vendor and product IDs are 16-bit numbers, not strings (VIA represents them as strings because you can't use the 0x notation in JSON), so it's case insensitive anyway.

Reply:
that makes sense! thanks. the correction has been made.